list.json

```diff
@@ -0,0 +1 @@
+[{"name":"justin"}]
```

Comment:
Database contents should not be stored in git, high potential for leaking secrets here!